activiti-image-generator/.gitignore

```diff
@@ -0,0 +1,2 @@
+/target
```

Comment:
@daisuke-yoshimoto we might need to remove it. As far as I remember we just need one .gitignore file in root directory.